x-pack/plugins/observability_solution/inventory/.storybook/get_mock_inventory_context.tsx

```diff
@@ -42,7 +42,7 @@ export function getMockInventoryContext(): InventoryKibanaContext {
         },
       },
     } as unknown as SharePluginStart,
-    telemetry: {} as unknown as ITelemetryClient,
+    telemetry: {} as unknown as TelemetryClient,
```

Comment:
Here should be referencing the interface, not the class. Following his [SOLID principle](https://www.digitalocean.com/community/conceptual-articles/s-o-l-i-d-the-first-five-principles-of-object-oriented-design#dependency-inversion-principle)

Reply:
https://github.com/elastic/kibana/pull/197055/commits/7a98daa2fe621e03e52d59f3cb43bb03fc826c7f